docs/src/main/paradox/release-notes/10.2.x.md

```diff
@@ -1,5 +1,13 @@
 # 10.2.x Release Notes
 
+## 10.2.4
+
+This release is essentially a security related patch release with a fix disallow parsing of Transfer-Encoding other than chunked [#3754](https://github.com/akka/akka-http/pull/3754)
```

Comment:
```suggestion
This release is a security-related patch release with a fix disallow parsing of Transfer-Encoding other than chunked [#3754](https://github.com/akka/akka-http/pull/3754)
```

Should we also add a note to https://doc.akka.io/docs/akka-http/snapshot/security.html ? We should probably include a link to https://nvd.nist.gov/vuln/detail/CVE-2021-23339 somewhere?